unlock-app/src/components/interface/locks/Settings/elements/SettingEmail.tsx

```diff
@@ -0,0 +1,37 @@
+import { EmailTemplatePreview } from './EmailTemplatePreview'
+import { SettingCard } from './SettingCard'
```

Comment:
Hum, Can you move the templates to a new sub package `@unlock-protocol/email-templates` that can then be included here and in wedlock? This way we don't have duplicated stuff that's going to be VERY hard to maintain.

Reply:
here is the PR: https://github.com/unlock-protocol/unlock/pull/10980